piecewise_web/index.html

```diff
@@ -239,7 +240,7 @@ <h3 class="league-gothic">Thank you!</h3>
 
 	function getNdtServer () {
 		var xhr = new XMLHttpRequest(),
-			mlabNsUrl = 'https://mlab-ns.appspot.com/ndt?format=json';
+			mlabNsUrl = 'https://mlab-ns.appspot.com/ndt_ssl?format=json';
```

Comment:
@nkinkade -- it'd be great if you could look at this specifically. we did this to get it working, but I don't know enough about mlab-ns options to know if this is the right option